lib/school_house_web/templates/error/404.html.eex

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
We can most likely make this `error.html` for all generic error messages and rely on the view itself to provide the specific message.